akita_inu_asa_utils.py

```diff
@@ -173,6 +196,25 @@ def create_app_signed_txn(private_key,
     return signed_txn, signed_txn.transaction.get_txid()
 
 
+def update_app_signed_txn(private_key,

```

Comment:
indicate in comment that function returns tuple of signed_txn and txid

same for other app functions here